miranda/structure/_structure.py

```diff
@@ -189,9 +159,9 @@ def structure_datasets(
     ----------
     input_files: str or Path or list of str or Path or GeneratorType
     output_folder: str or Path
-    project: {"cordex", "cmip5", "cmip6", "isimip-ft", "reanalysis", "pcic-candcs-u6"}, optional
+    activity: {"cordex", "cmip5", "cmip6", "isimip-ft", "reanalysis", "pcic-candcs-u6"}, optional
```

Comment:
This probably why I think we need to keep project around. There are incompatible metadata differences between CMIP5-CORDEX (and probably CMIP6-CORDEX when it'll be produced), CMIP5/6, CMIP5-ISIMIP, etc.

I can't think of a better way to specify the conventions to use / expectations of fields when decoding a particular file from a given project. The changes here probably need to be stepped back.

`reanalysis` here is for things that have been collected and converted, therefore they already have all the expected fields. This will be the case for station data, and any other data that gets converted with Miranda. Maybe something like `converted`/`compliant`?